notebooks/official/training/hyperparameter_tuning_tensorflow.ipynb

```diff
@@ -665,7 +620,7 @@
       "source": [
         "### Write Dockerfile\n",
         "\n",
-        "After writing your training code, you create a Dockerfile. In the Dockerfile, you include all the commands needed to run the image. It'll install all the necessary libraries, including the CloudML Hypertune library, and set up the entry point for the training code."
+        "After writing your training code, you create a Dockerfile. In the Dockerfile, you include all the commands needed to run the image. It install's all the necessary libraries, including the CloudML Hypertune library, and set up the entry point for the training code."
```

Comment:
delete second "you" in second sentence:
- In the Dockerfile, include all the commands needed to run the image
Change "install's" to "installs"